orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/RetryableIgorTask.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.igor.tasks;
+
+import com.google.common.collect.ImmutableMap;
+import com.netflix.spinnaker.orca.ExecutionStatus;
+import com.netflix.spinnaker.orca.RetryableTask;
+import com.netflix.spinnaker.orca.TaskResult;
+import com.netflix.spinnaker.orca.pipeline.model.Stage;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import retrofit.RetrofitError;
+
+import javax.annotation.Nonnull;
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+@RequiredArgsConstructor
+@Slf4j
+public abstract class RetryableIgorTask implements RetryableTask {
+  public long getBackoffPeriod() {
+    return TimeUnit.SECONDS.toMillis(2);
```

Comment:
Are these worth configuring? How do we handle Clouddriver tasks?

Reply:
Prior to this, it was just hard-coded in `MonitorJenkinsJobTask` (though I realize I slightly shortened the retry period, which I'll fix since I have no good reason to shorten it).

I don't think there's a lot of value in allowing this to be configurable...if someone had a good use case for why it should be different (and we shouldn't just change it for everyone) then I'd consider it, but absent that I don't think it's worth adding more config fields.

Reply:
Sounds good.